### PR TITLE
IBX-4857: Changed help output for ibexa:reindex command

### PR DESCRIPTION
**Pull request by papcio122**
| Question                                  | Answer
| ---------------------------------------- | ------------------
| **JIRA issue**                          | [IBX-4857](https://issues.ibexa.co/browse/IBX-4857)
| **Type**                                   | improvement
| **Target Ibexa version** | `v3.3` `v4.3`
| **BC breaks**                          | no

### Description
Improvement for help output of ibexa:reindex.

#### Checklist:
- [x] Provided PR description.
- [x] Tested the solution manually.
- [ ] Provided automated test coverage.
- [ ] Checked that target branch is set correctly (master for features, the oldest supported for bugs).
- [x] Ran PHP CS Fixer for new PHP code (use `$ composer fix-cs`).
- [ ] Asked for a review (ping `@ezsystems/engineering-team`).

### CODE REVIEW

File: eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php [hunk 1]
```diff
@@ -158,7 +158,7 @@ protected function configure()
                     - Refresh (add/update/remove) index on a set of content ID's:
                       <comment>ibexa:reindex --content-ids=2,34,68</comment>
                     
-                    - Refresh (add/update) index of a subtree:
+                    - Refresh (add/update) index of a subtree for a single location ID:
```
Comment (konradoboza) [inline eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php]: Maybe:
```suggestion
                    - Refresh (add/update) index of a subtree for a single Location ID:
```
or `locationId`.

**Review by konradoboza — COMMENTED** [review]
Congratz on your first contribution to the Kernel @papcio122! 😉

**Review by konradoboza — APPROVED** [review]

File: eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php [hunk 2]
```diff
@@ -158,7 +158,7 @@ protected function configure()
                     - Refresh (add/update/remove) index on a set of content ID's:
                       <comment>ibexa:reindex --content-ids=2,34,68</comment>
                     
-                    - Refresh (add/update) index of a subtree:
+                    - Refresh (add/update) index of a subtree for a single Location ID:
```
Comment (alongosz) [inline eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php:161]: @ezsystems/documentation-team does this sound good to you as well?

**Review by adamwojs — APPROVED** [review]

**Review by Steveb-p — APPROVED** [review]

**Review by DominikaK — APPROVED** [review]

**Comment by alongosz** [comment]
Merged. No QA required.

**Pull request merged**